src/argilla/client/feedback/schemas/remote/shared.py

```diff
@@ -21,6 +21,7 @@
 
 
 class RemoteSchema(BaseModel, ABC):
+    # TODO(@alvarobartt): Review optional id configuration for remote schemas
```

Comment:
It's optional for the moment, as we're not using those neither for the `fields` nor for the `questions`, but sure, we should make it mandatory once it's fully covered, as well as the `client`